### PR TITLE
manage members of a team

### PR DESCRIPTION
**Pull request by sethvincent**
This pr allows team team moderators to add and remove members of a team.

### CODE REVIEW

File: components/add-member-form.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Formik, Field, Form, ErrorMessage } from 'formik'
+
+function Button ({ type, disabled }) {
```
Comment (sethvincent) [inline components/add-member-form.js]: I'd like to reconcile this with the existing `Button` component. It could happen in this pr or a follow up.

Comment (kamicut) [reply components/add-member-form.js]: Ah yeah I couldn't decide if these should be different buttons. I couldn't get the disabled behavior, I just colored the button differently. You can see it in my PR

Comment (sethvincent) [reply components/add-member-form.js]: let's deal with this in a follow up pr.

File: pages/team.js [hunk 2]
```diff
@@ -66,6 +115,27 @@ export default class Team extends Component {
     const moderators = map(prop('osm_id'), team.moderators)
     const isUserModerator = contains(parseInt(this.props.user.uid), moderators)
 
+    let members = team
```
Comment (kamicut) [inline pages/team.js]: I think you mean `team.members`?
```suggestion
    let members = team.members
```

**Comment by kamicut** [comment]
I like the popup!

**Pull request merged**